dcurve/package.json

```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "@dgrants/contracts": "^0.0.1",
     "@dgrants/utils": "^0.0.1",
+    "axios": "^0.21.1",
```

Comment:
IMO we should use the built-in [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) API instead of axios, unless we need some of the fancier axios features (which it seems we don't), because:
- fewer dependencies -> smaller bundle -> better performance
- fewer dependencies -> less attack surface